core/io/stream_peer_gzip.cpp

```diff
@@ -76,6 +76,7 @@ Error StreamPeerGZIP::start_decompression(bool p_is_deflate, int buffer_size) {
 
 Error StreamPeerGZIP::_start(bool p_compress, bool p_is_deflate, int buffer_size) {
 	ERR_FAIL_COND_V(ctx != nullptr, ERR_ALREADY_IN_USE);
+	ERR_FAIL_COND_V_MSG(buffer_size <= 0, ERR_INVALID_PARAMETER, "provided buffer_size is invalid. try setting buffer_size > 0");
```

Comment:
```suggestion
	ERR_FAIL_COND_V(buffer_size <= 0, ERR_INVALID_PARAMETER);
```
Message is clear enough IMO, but if it should be kept it should be:
```suggestion
	ERR_FAIL_COND_V_MSG(buffer_size <= 0, ERR_INVALID_PARAMETER, "Provided buffer_size is invalid. Try setting buffer_size > 0.");
```
Or similar